Meadowlark-js/package.json

```diff
@@ -59,6 +59,8 @@
     "test:e2e:debug": "cross-env DEBUG=testcontainers* npm run test:e2e:jest",
     "test:e2e:build": "npm run docker:build && npm run test:e2e:jest",
     "test:e2e:build:wsl1": "npm run docker:build:wsl1 && npm run test:e2e:jest",
+    "test:e2e:dev:setup": "cross-env TESTCONTAINERS_RYUK_DISABLED=true ts-node tests/e2e/setup/DevModeConfig.ts",
+    "test:e2e:dev:exit": "docker stop mongo-test opensearch-test meadowlark-api-test && docker rm -f mongo-test opensearch-test meadowlark-api-test && docker network prune -f",
```

Comment:
The network pruning only affects unused networks, if not done, there would be many docker networks stored without use